includes/Services/LabkiPackManager.php

```diff
@@ -495,11 +506,20 @@ public function installSinglePack(
 		?array $rewriteMap = null,
 		?array $manifestPages = null
 	): array {
-		$packName = $packDef['name'];
-		$version = $packDef['version'];
-		$pages = $packDef['pages'];
+		$packName = $packDef['name'] ?? '(unnamed)';
```

Comment:
packs must have a name by virtue of a key being required in the `packs` object. we should throw an error if we don't have one.